WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -15,12 +15,42 @@ final class DashboardViewController: UIViewController {
     private let dashboardUIFactory: DashboardUIFactory
     private var dashboardUI: DashboardUI?
 
+    private var isDefaultSiteNameAvailable: Bool {
+        return ServiceLocator.stores.sessionManager.defaultSite?.name != nil && ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles)
+    }
+
+    private var titleName: String {
+        if !isDefaultSiteNameAvailable {
+            return ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.title
+        }
+        return Localization.title
+    }
+
     // MARK: Subviews
 
     private lazy var containerView: UIView = {
         return UIView(frame: .zero)
     }()
 
+    private lazy var storeNameLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 1
```

Comment:
I think this line is not needed as it is the default!

Reply:
Yep xD